pandasgui/widgets/grapher.py

```diff
@@ -155,7 +155,7 @@ def clear_layout(layout):
 
 def line(**kwargs):
     key_cols = []
-    for arg in [a for a in ['x', 'color', 'line_dash', 'line_group', 'facet_row', 'facet_col'] if a in kwargs.keys()]:
+    for arg in [a for a in ['x', 'color', 'line_dash', 'line_group', 'hover_name', 'facet_row', 'facet_col'] if a in kwargs.keys()]:
```

Comment:
Regarding intent here for having `hover_name` but not putting `marker_symbol` in the above list: if `hover_name` is specified same as `marker_symbol`, this will be part of the `groupby` mean, which is what one would want for a proper time series, but not always for time dependent line plots (ie, _age, experiment, voltage sweep_) where we might be interested in the cohort average behaviour. Specifying `marker_symbol` only will accomplish that, while specifying `marker_symbol` AND `hover_name` (or `line_group` would work too if not already in use) to the same variable will cause this to be included in the group by.